.azure-pipelines/windows.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - script: |
+      :: Use Windows CRLF line endings for checked-out text files
+      git config --global core.autocrlf true
+      set
+    displayName: Print environment variables
+  - checkout: self
+    fetchDepth: 1
+  - script: |
+      echo on
+      call "%VSINSTALLDIR%\VC\Auxiliary\Build\vcvarsall.bat" %ARCH%
+      FOR /F "tokens=* USEBACKQ" %%F IN (`where cl.exe`) DO (SET MSVC_CC=%%F)
+      FOR /F "tokens=* USEBACKQ" %%F IN (`where lib.exe`) DO (SET MSVC_AR=%%F)
+      FOR /F "tokens=* USEBACKQ" %%F IN (`where ld.exe`) DO (SET MSVC_LD=%%F)
```

Comment:
You could use `%%~fsF` to get the short names without spaces. That might simplify a bit of the script.

Reply:
BTW: LD is not used anywhere, it would have to be `link.exe` anyway.